src/main/java/edu/kit/ipd/crowdcontrol/objectservice/database/operations/NotificationOperations.java

```diff
@@ -122,60 +137,80 @@ public boolean delteNotification(int notificationID) {
      * @return {@code true} if deleted, {@code false} otherwise
      */
     public boolean deleteNotification(int id) {
-        NotificationRecord record = create.newRecord(NOTIFICATION);
-        record.setIdNotification(id);
-
-        return create.executeDelete(record, NOTIFICATION.ID_NOTIFICATION.eq(id)) == 1;
+        create.deleteFrom(NOTIFICATION_RECEIVER_EMAIL)
+                .where(NOTIFICATION_RECEIVER_EMAIL.NOTIFICATION.eq(id));
```

Comment:
they delete themselves if the notification gets deleted. same with Notification_token